src/app/Header/components/MenuRight/Login/Login.jsx

```diff
@@ -16,7 +16,7 @@ const LoginButton = styled(Button)`
   margin: 50px auto 28px;
 `;
 
-export default function Login({ onClose }) {
+export default function Login({ onClose, setShowForgetPasswordModal }) {
```

Comment:
这个和 @hachilam92 @JavaScriptN0ob pair 过了吗？
```suggestion
export default function Login({ onClose, onShowForgetPassword }) {
```

Reply:
没跟我pair，不过我看他已经复用了子维 useToggler

Reply:
嗯 下午跟ziwei pair了一下